__DEFINES/ruins.dm

```diff
@@ -0,0 +1,87 @@
+#define RUINTYPE_SPACE "Space"
+#define RUINTYPE_LAVA "Lava"
+#define RUINTYPE_SNOW "Snow"
+#define RUINTYPE_SAND "Sand"
+#define RUINTYPE_JUNGLE "Jungle"
+#define RUINTYPE_ROCK "Rock"
+#define RUINTYPE_BEACH "Beach"
+#define RUINTYPE_WASTE "Waste"
+#define RUINTYPE_YELLOW "Yellow"
+#define RUINTYPE_XENO "Xeno"
+#define RUINTYPE_DESERT "Desert"
+#define RUINTYPE_WATER "Water"
+#define RUINTYPE_BATTLEFIELD "Battlefield"
+
+/// do not actually use this for your ruin type, this is for the ruintype_to_list proc
+#define RUINTYPE_EVERYTHING "Everything"
+
+// HEY LISTEN!
+// IF YOU ADD NEW A NEW RUIN TYPE
+// PLEASE MAKE SURE YOU ADD IT TO THE BELOW LIST AND PROC!
+
+#define RUINTYPE_LIST_ALL list(\
```

Comment:
Why not use a global list ? In fact I'm pretty sure it'd be a lot more efficient to have one global list (or a `static` list on a datum) shared by all procs than just instantiate a new list everytime you need it. You don't appear to even use this ?

Reply:
I ported this blindly; upon further review this entire file really isn't necessary as it was just used to catalogue the many ruins on Shiptest.